pkg/scheduler/api/node_info.go

```diff
@@ -281,23 +281,30 @@ func (ni *NodeInfo) setNodeGPUInfo(node *v1.Node) {
 	if node == nil {
 		return
 	}
-	memory, ok := node.Status.Capacity[VolcanoGPUResource]
-	if !ok {
-		return
-	}
-	totalMemory := memory.Value()
 
-	res, ok := node.Status.Capacity[VolcanoGPUNumber]
+	res, ok := node.Status.Capacity[GPUResourceName]
```

Comment:
https://github.com/volcano-sh/devices/ will use `VolcanoGPUNumber` as resource name. If change to `GPUResourceName` how to work with volcano-sh/devices